kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -185,45 +187,109 @@ public void after() throws Exception {
   }
 
   @Test
-  public void testFilesWrittenToBucketAvro() throws Throwable {
+  public void testBasicRecordsWrittenAvro() throws Throwable {
     //add test specific props
     props.put(FORMAT_CLASS_CONFIG, AvroFormat.class.getName());
-    testBasicRecordsWritten(AVRO_EXTENSION);
+    testBasicRecordsWritten(AVRO_EXTENSION, /*addExtensionInTopic=*/false);
```

Comment:
Regarding the argument comments, IDEs would usually do this for you. Is it supposed to help reading the code outside IDEs?

Reply:
yes, for example, on a github review :)

Reply:
btw another added benefit is that if the actual argument name changes, the IDE will flag it at the call site.  So, for example, with a bunch of booleans in a row, if they get mixed up (i.e. someone inserts a boolean in the middle), then if the argument name does not match the /*xxx=*/ comment, then the IDE will highlight it as a problem.

Reply:
You would need to have IDE support to update these during refactors as well :/ 
Are we doing this in other repositories at Confluent? 
I don't have a strong preference, but if this is a first it would probably need a bigger discussion.

Reply:
I think the whole idea is that you wouldn't want the IDE to refactor it automatically.  It's just a comment.

Reply:
removed anyway